schema/omega-services.async.api.yaml

```diff
@@ -777,6 +837,195 @@ components:
+        asset-legal-status:
+          $ref: '#/components/schemas/LegalStatusSummary'
+        legacy-tna-cs13-record-type:
+          $ref: '#/components/schemas/LegacyTNACS13RecordType'
+        designation-of-edition:
+          type: string
+          description: Description identifying an edition to which a record (e.g. Map) belongs.
+        created:
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/TemporalInstant'
+            - $ref: '#/components/schemas/TemporalInterval'
+          description: The period of time or instant during which the record itself was created
+        covering:
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/TemporalInstant'
+            - $ref: '#/components/schemas/TemporalInterval'
+          description: The period of time or instant that the subject matter of the record spans
+        archivists-note:
+          $ref: '#/components/schemas/GeneralDatedDescription'
+          description: An archivists note about the record
+        source-of-acquisition:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: The URI of an agent
+        custodial-history:
+          type: string
+        adminstrative-biographical-background:
+          type: string
+        accumulation:
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/TemporalInstant'
+            - $ref: '#/components/schemas/TemporalInterval'
+          description: The period of time or instant during which the record was accumulated
+        appraisal:
+          type: string
+        accrualPolicy:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: The URI of the accrual policy of the record
+        layout:
+          type: string
+          description: Additional textual information adding context around the arrangement/layout of the record
+        publication-note:
+          $ref: '#/components/schemas/GeneralDatedDescription'
+          description: A note about the publication of the record
+        referenced-by:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralLabelledIdentifier'
+          description: Links to other resource which reference this record
+        related-to:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralLabelledIdentifier'
+          description: Links to related material
+        separated-from:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralLabelledIdentifier'
+          description: Links to separated material
+        subject:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: Links to subjects. Typically these are URI of Agent, Person, or Location entities. They may also be URI of terms from a controlled vocabulary
```

Comment:
The `dct:subject` can also be a bNode such as:
```
dct:subject [ 
    dct: description "from 1965" ;
    dct:subject cat:agent.24
] 
```
Meaning that this should perhaps be an object? Although, worth noting that we still have a ticket open relating to dct:subject: https://national-archives.atlassian.net/browse/PACT-962

Reply:
@rwalpole Good spot! I have now amended this to support the description as well as the identifier